pushtest/client.py

```diff
@@ -35,8 +37,12 @@ def hello(self):
             chans = self.channels.keys()
         else:
             chans = []
-        msg = json.dumps(dict(messageType="hello", uaid=self.uaid or "",
-                              channelIDs=chans))
+        if self.use_webpush:
+            msg = json.dumps(dict(messageType="hello", uaid=self.uaid or "",
+                                  use_webpush=True, channelIDs=chans))
```

Comment:
This seems buried and might be a problem if we add more options. Does it make sense to have a common dict we add to rather than duplicate it with the modification?